pkg/plugins/generic/generic_plugin.go

```diff
@@ -264,84 +273,83 @@ func needDriverCheckVdpaType(state *sriovnetworkv1.SriovNetworkNodeState, driver
 }
 
 // setKernelArg Tries to add the kernel args via ostree or grubby.
-func setKernelArg(karg string) (bool, error) {
+func setKernelArg(helper helper.HostHelpersInterface, mode, karg string) error {
 	log.Log.Info("generic plugin setKernelArg()")
```

Comment:
not strictly related to this PR, but it would be useful to log the kernel argument we are trying to add:
```
log.Log.Info("generic plugin setKernelArg()", "karg", karg)
```